spec.bs

```diff
@@ -702,6 +707,13 @@ To <dfn>validate and convert auction ad config</dfn> given an {{AuctionAdConfig}
     a {{TypeError}}.
   1. [=map/Set=] |auctionConfig|'s [=auction config/per buyer priority signals=][|buyer|] to
     |signals|.
+1. If |config|["{{AuctionAdConfig/perBuyerCurrencies}}"] [=map/exists=], [=map/for each=]
+  |key| → |value| of |config|["{{AuctionAdConfig/perBuyerCurrencies}}"]:
+  1. If the result of result of [=checking whether a string is a valid currency tag=] on |value| is false, [=exception/throw=] a {{TypeError}}.
+  1. If |key| equals to "*", then set |auctionConfig|'s

```

Comment:
```suggestion
  1. If |key| is "*", then set |auctionConfig|'s

```

Reply:
Tries to keep brain from going on tangent on nature of equality and mumbling something about Leibnitz: done.

Reply:
Actually the other fields with this syntax use equals to as well, should I change them?

Reply:
Yeah, if you don't mind (sorry if that's annoying!)

Reply:
And done on this, since it seems like this may land earlier...